docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -418,53 +350,18 @@ Let s199 be the proportion of SAM with oedema among those with SAM
     - Note
   * - :math:`r3_{ux}`
     - transition probability
-    - recovery probability of untreated MAM among 0-6 months
+    - Daily recovery probability of untx MAM 
+    - :math:`\frac{1}{\text{median time-to-recovery of utx MAM}}` 
```

Comment:
I'm not sure why these are called effect sizes -- it seems like they are just the definitions of the outcomes (e.g. `r3_ux` equals 1/(median time to recovery of untreated MAM, in days), correct?). Perhaps it will make more sense once you finish updating this section?

Reply:
ah, you are right. I will update the table